app/aws/AuditTrailDB.scala

```diff
@@ -108,11 +59,21 @@ object AuditTrailDB {
     queryResult(dynamoDB, request)
   }
 
+  private def attrCondition(
```

Comment:
I'm not 100% convinced this helper is worthwhile but if it is perhaps we could call it something like `attrEqualCondition`, to better convey that this is hard-coded to do an EQ check?

There are only two uses of this helper, and its definition isn't loads shorter than its invocation. Would it be simpler to write the builder out each time, especially since this would be consistent with the "BETWEEN" condition used just below for date range?

If IntelliJ did that underlined "Duplicate code fragment" thing then no worries, that's a clear sign wiser minds than I say this is definitely worthwhile 😁

Reply:
Yes this did come from an Intellij prompt!  I'll have another look at this one as well

Reply:
Renamed in 21e2630e7fce7a6189807f151bbc061a39af9b2b